### ISSUE TITLE
[Bug]: Manual Import Issues (Periods Within Filename & Cleanup)

### ISSUE THREAD

**Issue by PhuriousGeorge** [tubearchivist/tubearchivist#311]
### Latest and Greatest

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).

### Operating System

Unraid

## Describe the bug
1. Any attempted import of offline (assumed to affect online too - untested) videos with periods `.` within the names causes an error

2. Manual Import of an online file errored out during cleanup phase per Discord discussion. EDIT: After much use, this only occurs with videos imported without thumbnails
 
## Steps To Reproduce
1. Attempted with multiple example files.  Once periods are removed, import works as expected.
2. Multiple attempts with many files

### Relevant log output

```shell
For Offline Import:
File names:
`6061._com - 20171030 - Pulse_TIG_Welding_-_Engrave_Your_Tools [YPL7pgYzzEY].mp4`
`6061._com - 20171030 - Pulse_TIG_Welding_-_Engrave_Your_Tools [YPL7pgYzzEY].webp`
`6061._com - 20171030 - Pulse_TIG_Welding_-_Engrave_Your_Tools [YPL7pgYzzEY].info.json`

[2022-08-19 21:25:38,496: WARNING/ForkPoolWorker-1] starting media file import
[2022-08-19 21:25:38,499: WARNING/ForkPoolWorker-1] {'media': False, 'video_id': False, 'metadata': '/cache/import/6061._com - 20171030 - Pulse_TIG_Welding_-_Engrave_Your_Tools [YPL7pgYzzEY].info.json', 'thumb': False, 'subtitle': []}: no matching media file found.
[2022-08-19 21:25:38,500: ERROR/ForkPoolWorker-1] Task home.tasks.run_manual_import[a6dabdb7-fbb2-4732-8bbc-c4bedfda7586] raised unexpected: ValueError()
Traceback (most recent call last):
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 153, in run_manual_import
    ImportFolderScanner().scan()
  File "/app/home/src/index/filesystem.py", line 189, in scan
    self.process_videos()
  File "/app/home/src/index/filesystem.py", line 256, in process_videos
    raise ValueError
ValueError```

Online Import:
```[2022-08-19 21:28:19,350: ERROR/ForkPoolWorker-1] Task home.tasks.run_manual_import[a7eb62c9-eb03-4f4c-9cc8-1f77ade573a9] raised unexpected: TypeError('remove: path should be string, bytes or os.PathLike, not bool')
Traceback (most recent call last):
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 153, in run_manual_import
    ImportFolderScanner().scan()
  File "/app/home/src/index/filesystem.py", line 189, in scan
    self.process_videos()
  File "/app/home/src/index/filesystem.py", line 264, in process_videos
    ManualImport(current_video, self.CONFIG).run()
  File "/app/home/src/index/filesystem.py", line 500, in run
    self._cleanup(json_data)
  File "/app/home/src/index/filesystem.py", line 562, in _cleanup
    os.remove(self.current_video["thumb"])
TypeError: remove: path should be string, bytes or os.PathLike, not bool```
```

**Comment by PhuriousGeorge** [comment]
To add, I would guess this is a similar filename parsing error:

File name: `Bryan_Lunduke - 20190201 - On_Static_Website_Generators_-_Hugo_and_Jekyll [HxxvzkgaiEo].info.json`

```[2022-08-20 01:09:45,262: ERROR/ForkPoolWorker-1] Task home.tasks.run_manual_import[8327ec8e-b20b-4d4c-a1ec-def35ffe3c56] raised unexpected: TypeError('remove: path should be string, bytes or os.PathLike, not bool')
Traceback (most recent call last):
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 153, in run_manual_import
    ImportFolderScanner().scan()
  File "/app/home/src/index/filesystem.py", line 189, in scan
    self.process_videos()
  File "/app/home/src/index/filesystem.py", line 264, in process_videos
    ManualImport(current_video, self.CONFIG).run()
  File "/app/home/src/index/filesystem.py", line 500, in run
    self._cleanup(json_data)
  File "/app/home/src/index/filesystem.py", line 562, in _cleanup
    os.remove(self.current_video["thumb"])
TypeError: remove: path should be string, bytes or os.PathLike, not bool```

**Comment by bbilly1** [comment]
both of these errors should be fixed in v0.2.2
